docs/source/channels.rst

```diff
@@ -0,0 +1,140 @@
+Shared Futures With Channels
+============================
+
+A channel is a changing stream of futures shared between any number of clients
+connected to the same scheduler.
+
+
+Examples
+--------
+
+Basic Usage
+~~~~~~~~~~~
+Create channels from your Client:
+
+.. code-block:: python
+
+    >>> client = Client('scheduler-address:8786')
+    >>> chan = client.channel('my-channel')
+
+Append futures onto a channel
+
+.. code-block:: python
+
+    >>> future = client.submit(add, 1, 2)
+    >>> chan.append(future)
+
+A channel maintains a collection of current futures added by both your
+client, and others.
+
+.. code-block:: python
+
+    >>> chan.futures
+    deque([<Future: status: pending, key: add-12345>,
+           <Future: status: pending, key: sub-56789>])
+
+If you wish to persist the current status of tasks outside of the distributed
+cluster (e.g. to take a snapshot in case of shutdown) you can copy a channel full
+of futures as it is a python deque_.
+
+.. _deque: https://docs.python.org/3.5/library/collections.html#collections.deque`
+
+.. code-block:: python
+
+    channelcopy = copy.deepcopy(chan.futures)
+
+You can iterate over a channel to get back futures.
+
+.. code-block:: python
+
+    >>> anotherclient = Client('scheduler-address:8786')
+    >>> chan = anotherclient.channel('my-channel')
+    >>> for future in chan:
+    ...     pass
+
+You can ensure that a channel is fully processed using the ``flush`` method
+
+.. code-block:: python
+
+    >>> client = Client('scheduler-address:8786')
+    >>> chan = client.channel('my-channel')
+    >>> future2 = client.submit(time.sleep,2)
+    >>> chan.append(future2)
+    >>> chan.flush()
+
+
+
+
+Example with local_client
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Using channels with local client allows for a more decoupled version
```

Comment:
Perhaps link here from `local_client` to http://distributed.readthedocs.io/en/latest/task-launch.html#submit-tasks-from-worker (or the appropriate in-document sphinx link as you do below with queues).  The casual reader may not know what "local client" means otherwise.